### PR TITLE
fix: add jina hub new type

### PR DESCRIPTION
**Pull request by rutujasurve94**
Updating README for new `--type` argument for `jina hub new`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ with Flow().add(uses=my_image_tag):
 git clone https://github.com/jina-ai/jina-hub.git
 
 pip install jina[hub]
-jina hub new
+jina hub new --type='pod'
```
Comment (hanxiao) [inline README.md:33]:
```suggestion
jina hub new --type pod
```

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ More Hub CLI usage can be found via `jina hub build --help`
 ## Work with Your Own Repository
 
 1. Create a new repository
-2. `pip install jina[hub] && jina hub new`
+2. `pip install jina[hub] && jina hub new --type='pod'`
```
Comment (hanxiao) [inline README.md:62]:
```suggestion
2. `pip install jina[hub] && jina hub new --type pod`
```

**Review by hanxiao — CHANGES_REQUESTED** [review]
single-quote is unnecessary

File: README.md (hunk 1 above)
Comment (rutujasurve94) [reply README.md:33]: Done

File: README.md (hunk 2 above)
Comment (rutujasurve94) [reply README.md:62]: Done

**Pull request merged**